README.md

```diff
@@ -262,6 +262,7 @@ Maintainers keep a folder with a clean checkout of the code and use [jenv.be](ht
 - make sure CI is green for the chosen commit
 - run `./gradlew releaseCheck`. If successful, a signed release will be at `collect_app/build/outputs/apk` (with an old version name)
 - verify a basic "happy path": scan a QR code to configure a new project, get a blank form, fill it, open the form map (confirms that the Google Maps key is correct), send form
+- run `./benchmark.sh` with a real device connected to verify performance
```

Comment:
@lognaturel I think it makes sense to define a specific phone here and that should probably be one you own.

Reply:
I primarily use a Samsung Galaxy A13. It seems I can't currently run these tests or any other I've tried because I always end up in the demo project and get an exception because the tests expect to be at the landing screen. They do work on an emulator.

Reply:
Sounds like it could be some incompatibility with how we reset state. We can have a look at it later today when we chat.